terraform/modules/windows-test-domain/scripts/WinRM/user_data.ps1

```diff
@@ -115,6 +115,6 @@ Enable-NetFirewallRule -DisplayGroup "Remote Desktop"
 Set-ItemProperty -Path 'HKLM:\System\CurrentControlSet\Control\Terminal Server\WinStations\RDP-Tcp' -name "UserAuthentication" -Value 1
 
 # Clone repos for bootstrap scripts
-$branch="main"
+$branch="domain-and-host-names"
```

Comment:
should we change this back to main before merging?

Reply:
Yes - I keep getting distracted. Doing that now.